mapie/tests/test_utils.py

```diff
@@ -508,3 +509,18 @@ def test_check_no_agg_cv_value_error(cv: Any) -> None:
         match=r"Allowed values must have the `get_n_splits` method"
     ):
         check_no_agg_cv(X_toy, cv, array)
+
+
+@pytest.mark.parametrize("X", [1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
```

Comment:
No need to test for so many values. Keep it simpler in this case. There is a choice to make between a bunch of tests and the time it takes to run them.